src/prefect/utilities/executors.py

```diff
@@ -10,7 +10,7 @@
 from concurrent.futures import TimeoutError as FutureTimeout
 from functools import wraps
 from logging import Logger
-from typing import TYPE_CHECKING, Any, Callable, Dict, List, Union
+from typing import TYPE_CHECKING, Any, Callable, Dict, List, Union, Callable, Optional
```

Comment:
Very minor, but it looks like `Callable` is being imported twice.

Reply:
good call. Actually, both are dead code... I'll do a last sweep through the imports.